src/main/resources/static/widget.js

```diff
@@ -0,0 +1,36 @@
+function getIframe() {
+  let tcTokenURL = new URL(document.currentScript.src).searchParams.get(
+    "tcTokenURL"
+  );
+  if (tcTokenURL === null) {
+    tcTokenURL = document.getElementById("useid-widget").dataset.tcTokenUrl;
+
+    if (tcTokenURL === undefined) {
+      const error = document.createElement("div");
+      error.innerHTML = "Fehlerhafte Konfiguration: TC Token nicht definiert.";
```

Comment:
The error should say "TC Token URL nicht definiert.", right?

Reply:
Right!